src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -2051,11 +2051,19 @@ struct HeatBalanceData : BaseGlobalStruct
     bool NoFfactorConstructionsUsed = true;
     bool NoCfactorConstructionsUsed = true;
     bool NoRegularMaterialsUsed = true;
+    bool DoLatentSizing = false; // true when latent sizing is performed during zone sizing
+    bool isAnyLatentLoad = false;
 
     Array1D<Real64> ZoneSNLoadHeatEnergy;
     Array1D<Real64> ZoneSNLoadCoolEnergy;
     Array1D<Real64> ZoneSNLoadHeatRate;
     Array1D<Real64> ZoneSNLoadCoolRate;
+    Array1D<Real64> ZoneLTLoadHeatEnergy;        // latent heating energy [J]
```

Comment:
All of these ZoneSN/ZoneLT arrays should be grouped in to a single struct (or mabye added to the existing `dataHeatBalance::AirReportVars` struct?).

Reply:
Noted.